WolvenKit.Interfaces/IWitcherArchive.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace WolvenKit.Interfaces
+{
+    public interface IWitcherArchive
```

Comment:
Should be renamed to IWitcherArchiveManager since it manages archive now it's not for the actual archives.